thunder/tests/test_networks.py

```diff
@@ -336,17 +337,32 @@ def test_quantization():
     model_fp_reference.requires_grad_(False)
     model_fp_reference.eval()
 
+    model_fp_reference2.set_kv_cache(1, device="cuda", dtype=torch.bfloat16)
+    model_fp_reference2.max_seq_length = 20
+    model_fp_reference2.requires_grad_(False)
```

Comment:
will it work if `requries_grad is True`?

Reply:
It doesn't mix with quantization, I'll add a separate test.